libs/server/Objects/Hash/HashObject.cs

```diff
@@ -228,7 +228,10 @@ public override unsafe void Scan(long start, out List<byte[]> items, out long cu
                 if (patternLength == 0)
                 {
                     items.Add(item.Key);
-                    items.Add(item.Value);
+                    if (!isNoValue)
```

Comment:
The count value will have to be adjusted as well if NOVALUES present as it is currently always multiplying it by two to account for key and value.

Reply:
Fixed